gui/poimanager/poimangui.py

```diff
@@ -626,6 +626,8 @@ def create_poi_from_click(self, button, pos):
         new_pos = self.poimanagerlogic().roi_origin
```

Comment:
This line is not needed anymore. Either initialize an empty array here or define the new position in one line like 
`new_pos = np.array([pos.x(), pos.y(), self.poimanagerlogic().scanner_position[2]])`